src/article.js

```diff
@@ -1,18 +1,11 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
-import "./css/App.css";
-
 class Article extends Component {
-  constructor(props) {
-    super(props);
-
-    this.handleArticleClick = this.handleArticleClick.bind(this);
-  }
-
-  handleArticleClick() {
+  handleArticleClick = () => {
```

Comment:
Here's a clever trick if you want to avoid writing these long constructors and binding every function

if you define your function as `handleArticleClick = () => { }` instead of `handleArticleClick() {}` the arrow function magically auto-binds the function to the class for you.